### ISSUE TITLE
New EX Folder Status: Pending Customer Action 

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#6311]
### Add 'Pending Customer Action' as a valid EX folder status.

**Stakeholder:** Tracy, Greg Pep.

**Acceptance Criteria:**
- 'Pending Customer Action' is an available folder status.
- No other automation or processes need to be built.

**Comment by TracyLinder** [comment]
@Nadin-Nader just fYI I put this on the sprint for June 30th. Not sure if there's room but we'll figure it out.

**Comment by SubhaShroff** [comment]
@Nadin-Nader
-  Is this just to add the status to EX Permit? 
What triggers the permit status change to 'Pending Customer Action'?

**Comment by Nadin-Nader** [comment]
Waiting on Greg Pepper's response:

> Hi Greg,
> 
> Hope you are doing well. I was reaching out to you regarding card # 6311 New EX Folder Status: Pending Customer Action. I have Sree from Launch IT looking into it and found that she had some questions on this request. 
> 
> We have the Acceptance Criteria as:
> 
> Acceptance Criteria:
> 
> 'Pending Customer Action' is an available folder status for the EX Folder.
> No other automation or processes need to be built.
> 
> We would like to know:
> 1)	What triggers the permit status change to 'Pending Customer Action'?
> 2)	Will this status change/add a date? and where?
> 3)	Will this status trigger anything?
> 4)	Is this status just a place holder that will eventually (6months to a year later) have any automation?
>

**Comment by Nadin-Nader** [comment]
@SubhaShroff  here is the Email response from Greg Prepper:


We would like to know:
1)	What triggers the permit status change to 'Pending Customer Action'?
**a.	Nothing for now. This is prepping for future enhancement but is needed to be added for manual input of folders for now. This is a temp work around for CPO permitting that will allow us to “Sand Bag” permits and not affect the current reporting of performance metrics for ROW staff. **

2)	Will this status change/add a date? and where?
**a.	No. All dates will stay the same.**

3)	Will this status trigger anything?
**a.	Not now but will trigger action when Contractor Self Assignment goes live at a later date.
b.	It will also be used for the future enhancement of Traffic Control Review when a Request for More Info action is taken on permits. (planned future state)**

4)	Is this status just a place holder that will eventually (6months to a year later) have any automation?
**a.	Yes. But it is an immediate need and priority for CPO permitting of upcoming CIP projects. This addition will allow us to be more proficient with our CIP permitting process and not negatively affect the ROW permitting stats on current reports. As is, we are affecting the performance metrics and also allow the elimination of  doubling staff work to input our CIP permits when the system closes open permits by batch. The Pending Customer Action status is not within that batch validation, so it will bypass closing our permits and doubling work of CPO and ATD staff. 
Killing 3 birds with one 1 status. 😊 Thanks!!**

**Comment by SubhaShroff** [comment]
@Nadin-Nader Pending Customer Action Status is added to EX on localhost.  I did the commit for this code change - 6a72d0599438

![image](https://user-images.githubusercontent.com/51680715/123836273-21184280-d8cf-11eb-906c-dbd8eebc17e5.png)

**Comment by SubhaShroff** [comment]
@Nadin-Nader DEV deployment request is submitted for commit:6a72d0599438- **SCTASK0148142**

**Comment by SubhaShroff** [comment]
### Status:
DEV deployment is done. I need to test it.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  DEV deployment is done. And here is how it looks on DEV:

![image](https://user-images.githubusercontent.com/51680715/123973187-2afa7e00-d981-11eb-8e57-00d8e6da0737.png)

### Status:
Ready for DEV review.

**Comment by Nadin-Nader** [comment]
@SubhaShroff lets go ahead and push it to TEST

**Comment by SubhaShroff** [comment]
@Nadin-Nader  TEST deployment request is submitted-**SCTASK0148690**

**Comment by SubhaShroff** [comment]
@Nadin-Nader  TEST deployed and I tested it. Kindly let me know if we can move it to PROD.

![image](https://user-images.githubusercontent.com/51680715/124136253-34035200-da4a-11eb-8747-41da6e6d79f0.png)

**Comment by Nadin-Nader** [comment]
@SubhaShroff lets get this to PROD.

## **APPROVED for PROD Deployment🎉🎉**

**Comment by SubhaShroff** [comment]
@Nadin-Nader PROD deployment request is submitted-**SCTASK0148905**

**Comment by Nadin-Nader** [comment]
**Status:**
Waiting on DevOps CM Team to deploy to PROD **SCTASK0148905**

**Comment by SubhaShroff** [comment]
@Nadin-Nader PROD deployment is done.

**Comment by Nadin-Nader** [comment]
On PROD and sent Greg P an email letting him know it is available for use. I will close the card if do not hear back from him tomorrow with issues or concerns by lunch:

Hi Greg,
Hope you are doing well. I wanted to let you know that the New EX Folder Status: Pending Customer Action is now on PROD. Please log on and verify. I will go ahead and close the card tomorrow by lunch if I do not hear from you with any issues or changes.

Respectfully,

Nadin A. Nader (He/Him/His)
IT Business Systems Analyst
Data and Technology Services
————————————————————————————
transportation.data@austintexas.gov 🚦 austinmobility.io

**Comment by Nadin-Nader** [comment]
Reply from Greg P.:

> From: Pepper, Gregory <Gregory.Pepper@austintexas.gov> 
> Sent: Friday, July 9, 2021 8:02 AM
> To: Nader, Nadin <Nadin.Nader@austintexas.gov>
> Cc: Linder, Tracy <Tracy.Linder@austintexas.gov>
> Subject: Re: New EX Folder Status: Pending Customer Action on PROD
> 
> Hey Nadin, 
> Thanks for the update.  I am out of office until Monday.  I will have to check on it when I get back. Thanks Again!!
> GPepper
> Sent via the Samsung Galaxy S20+ 5G, an AT&T 5G smartphone
> Get Outlook for Android

I will follow up with him today to see if all is well.

**Comment by Nadin-Nader** [comment]
Both Greg and Jessica said this was good to go. Closing.

> From: Davis, Jessica <Jessica.Davis@austintexas.gov> 
> Sent: Tuesday, July 13, 2021 1:29 PM
> To: Nader, Nadin <Nadin.Nader@austintexas.gov>; Pepper, Gregory <Gregory.Pepper@austintexas.gov>
> Subject: Re: New EX Folder Status: Pending Customer Action on PROD
> 
> We are good!😁
> Sent from my T-Mobile 4G LTE Device
> Get Outlook for Android
> ________________________________________
> From: Pepper, Gregory <Gregory.Pepper@austintexas.gov>
> Sent: Tuesday, July 13, 2021 1:04:54 PM
> To: Nader, Nadin <Nadin.Nader@austintexas.gov>; Davis, Jessica <Jessica.Davis@austintexas.gov>
> Subject: RE: New EX Folder Status: Pending Customer Action on PROD 
>  
> Hey Jessica,
> Are we good on the prod status? Thanks!
>  
> Respectfully,
>  
> Gregory Pepper
> Construction & Inspection Services Program Manager
> Utility Coordination Program Manager
> AULCC Chairperson
> 512-974-7282 Office
> 512-277-4787 Mobile
